octave.m

```diff
@@ -0,0 +1,42 @@
+%numbers
```

Comment:
I believe we dropped octave.m support we should drop this file

Reply:
Thanks for reminding me about that! It has now been removed.